.husky/pre-commit

```diff
@@ -1 +1,2 @@
-yarn lint-staged
\ No newline at end of file
+yarn lint-staged
+yarn check-format
```

Comment:
```suggestion
yarn check-format
```
This will ensure unforamtted file

Reply:
No no, we want it to lint. The CI is just a fallback.